src/instrumenter/instrument.ts

```diff
@@ -243,6 +374,13 @@ function emitAssert(
                 factory.makeLiteral("<missing>", LiteralKind.String, "", `HIT: ${strMessage}`)
             ]);
         }
+    } else if (instrCtx.assertionMode === "hardhat") {
+        const strMessage = `000000:0000:000 ${annotation.id}: ${annotation.message}`;
```

Comment:
strMessage is already defined on line 362

Reply:
Addressed in 10c64ec